core/src/main/java/com/netflix/conductor/core/execution/DeciderService.java

```diff
@@ -68,12 +69,14 @@
 public class DeciderService {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(DeciderService.class);
+    static final String TASKS_MAX_LIMIT = "conductor.app.tasks.max.limit";
```

Comment:
Not a deal breaker, but this property could use kebab-case to maintain consistency with other property names.